packages/core/src/mappers/renderer.ts

```diff
@@ -196,9 +200,62 @@ export const createDefaultValue = (
     return extractDefaults(resolvedSchema, rootSchema);
   } else if (hasType(resolvedSchema, 'null')) {
     return null;
-  } else {
-    return {};
   }
+
+  let combinatorDefault = undefined;
+
+  combinatorDefault = createDefaultValueForCombinatorSchema(
```

Comment:
Instead of this additional method we could create a `doCreateDefaultValue` instead which behaves the same as the current one but returns `undefined` if there is no default value.

In `createDefaultValue` we then only call the `doCreateDefaultValue` and return `{}` instead of `undefined`.

This keeps backward compatibility but avoids the magic symbol

Reply:
sure will do

Reply:
fixed